src/analysis/raster/qgsninecellfilter.cpp

```diff
@@ -30,8 +41,135 @@ QgsNineCellFilter::QgsNineCellFilter( const QString &inputFile, const QString &o
 
 }
 
+// TODO: return an anum instead of an int
 int QgsNineCellFilter::processRaster( QgsFeedback *feedback )
 {
+#ifdef HAVE_OPENCL
+  if ( QgsOpenClUtils::enabled() && QgsOpenClUtils::available() && ! openClProgramBaseName( ).isEmpty() )
+  {
+    // Load the program sources
+    QString source( QgsOpenClUtils::sourceFromBaseName( openClProgramBaseName( ) ) );
+    if ( ! source.isEmpty() )
+    {
+      try
+      {
+        QgsMessageLog::logMessage( QObject::tr( "Running OpenCL program: %1" )
```

Comment:
I think this should be a QgsDebugMsg instead of a message log - it seems more of a debug/testing feedback to me